src/libs/actions/IOU.ts

```diff
@@ -10208,9 +10214,15 @@ function approveMoneyRequest(expenseReport: OnyxEntry<OnyxTypes.Report>, full?:
     const predictedNextState = isLastApprover(approvalChain) ? CONST.REPORT.STATE_NUM.APPROVED : CONST.REPORT.STATE_NUM.SUBMITTED;
     const managerID = isLastApprover(approvalChain) ? expenseReport.managerID : getNextApproverAccountID(expenseReport);
 
-    // TODO: Replace onyx.connect with useOnyx hook (https://github.com/Expensify/App/issues/66365)
-    // eslint-disable-next-line @typescript-eslint/no-deprecated
-    const optimisticNextStep = buildNextStep(expenseReport, predictedNextStatus);
+    const optimisticNextStep = buildNextStepNew({
+        report: expenseReport,
+        predictedNextStatus: CONST.REPORT.STATUS_NUM.OPEN,
```

Comment:
```suggestion
    const optimisticNextStep = buildNextStepNew({
        report: expenseReport,
        predictedNextStatus,
```

@DylanDylann is it intended or by mistake you set the incorrect `predictedNextStatus`(open instead of approved) in `approveMoneyRequest`?

Reply:
It is my mistake

Reply:
will fix here: https://github.com/Expensify/App/pull/73816

Reply:
Thanks 👍